### ISSUE TITLE
autoblock member requests from members who blocked all admins

### ISSUE THREAD

**Issue by rubinovitz** [kern/hackbot#23]
would be cool if possible and clean out member requests

**Comment by frankcash** [comment]
I'm j/c why this is necessary because as an admin you can still see the posts of these members.  Maybe they want to participate in a meaningful way but don't want to interact with mods outside of the group (just to play devil's advocate here).

**Comment by rubinovitz** [comment]
100% of the ones I've seen are bots.

On Wed, Sep 16, 2015 at 3:38 PM, Frank Cash notifications@github.com
wrote:

> I'm j/c why this is necessary because as an admin you can still see the
> posts of these members. Maybe they want to participate in a meaningful way
> but don't want to interact with mods outside of the group (just to play
> devil's advocate here).
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kern/hackbot/issues/23#issuecomment-140863398.

## 

Jennifer Rubinovitz

**Comment by frankcash** [comment]
oh I see

**Comment by rubinovitz** [comment]
we also have issues with the # of pending requests so it would be nice to
clear these fake ones out

On Wed, Sep 16, 2015 at 3:47 PM, Frank Cash notifications@github.com
wrote:

> oh I see
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kern/hackbot/issues/23#issuecomment-140867197.

## 

Jennifer Rubinovitz

**Comment by kern** [comment]
I don't believe the Graph API exposes member requests, but this could potentially be something we implement a scraper for.

**Comment by rubinovitz** [comment]
k megbot time probably
